apps/pie-docs/src/components/text-input/overview/overview.md

```diff
@@ -138,7 +143,7 @@ Use when a text input requires a card number to be entered, and reflect the card
 {% contentLayout %}
   {% contentItem %}
     <h4>Placeholder</h4>
-    <p>Placeholder text should not contain crucial information, and only displayed if beneficial to the user.</p>
+    <p>Placeholder text in the text input offers the user suggestions or examples for what to enter.</p>
```

Comment:
I think the previous message was a bit clearer in what not to do. Can I ask why we want to make this change?

Reply:
Discussed this with @tingting-lau and we decided to revert the change